js/content/store.js

```diff
@@ -912,6 +913,49 @@ class AppPageClass extends StorePageClass {
         });
     }
 
+    addFullscreenScreenshotView() {
+        function es_toggleFullScreen(event) {
+            if (!document.fullscreenElement) {
+                let element = event.target.closest(".screenshot_popup_modal_content");
+                element.requestFullscreen();
+            } else {
+                document.exitFullscreen();
+            }
+        }
+
+        function es_initFSVButton(event) {
+            if (event.animationName !== "es_screenshot_popup_modal_hook") return;
+
+            let modalFooter = document.querySelector(".screenshot_popup_modal_footer");
+            let fsvButton = document.createElement('div');
```

Comment:
Where applicable, we create HTML tags via parsing a template literal and then passing it to one of the functions of the `HTML` class